### PR TITLE
luxeradio footer section

### PR DESCRIPTION
**Pull request by cp-priyanka-g**
Create a Footer section having a background image which has a parallax effect.
Smile+ text having a link(presently page is not created so there is no link added).
CTA section with copyright text.

Add a dummy video in Video Section.
Add spacing in Element Section.

### CODE REVIEW

File: vue-frontend/src/components/home-new/CTASection.vue [hunk 1]
```diff
@@ -63,6 +66,16 @@
...
       </ul>
     </div>
+    <div
+      class="tw-text-center ... tw-mt-32"
+      :class="[
+        currentRoutePath == '/portfolio/' + id ? '' : 'hide-copyright-text',
+      ]"
+    >
+      <span class="v2-normal-3-text"
+        >2021 Canopas Software LLP. All rights reserved.</span
```
Comment (cp-shruti-s) [inline vue-frontend/src/components/home-new/CTASection.vue:76]: Add copyright symbol and can't show static years in copyright

**Comment by cp-shruti-s** [comment]
Screencast from 29-06-22 02_16_37 PM IST

parallax effect is not apply

**Comment by cp-shruti-s** [comment]
Screencast from 29-06-22 02_17_12 PM IST

Hide video player on hover effect

File: vue-frontend/src/components/portfolio/FooterSection.vue [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  },
+};
+</script>
+<style lang="scss" scoped>
+section.background-image {
+  transform: translateZ(-1px) scale(1.5);
+  z-index: -1;
+}
+
+.un {
```
Comment (cp-shruti-s) [inline vue-frontend/src/components/portfolio/FooterSection.vue:34]: Use tailwind css for this animation

File: vue-frontend/src/components/home-new/CTASection.vue [hunk 3]
```diff
@@ -1,78 +1,112 @@
...
+              We offer 100% Money-back guarantee. so, there’s no risk to you.
+            </div>
+          </li>
+        </ul>
       </div>
       <div
-        class="v2-title-3-text text-left v2-canopas-gradient-text reason-text mt-5"
+        class="tw-text-center ... tw-pt-16 tw-pb-32 xl:tw-pb-1"
+        :class="[
+          currentRoutePath == '/portfolio/' + id ? '' : 'hide-copyright-text',
```
Comment (cp-shruti-s) [inline vue-frontend/src/components/home-new/CTASection.vue:84]: Hide that whole `copyright div` not only text 
see the difference 

1. after add copyright ( CTA section in home page )
Screenshot from 2022-06-30 16-36-42

2. Before CTA section in home page 
Screenshot from 2022-06-30 16-36-20

File: vue-frontend/src/components/home-new/CTASection.vue [hunk 4]
```diff
@@ -1,78 +1,116 @@
...
+            <span class="bullet"></span>
+            <div>
+              We offer 100% Money-back guarantee. so, there’s no risk to you.
+            </div>
+          </li>
+        </ul>
       </div>
       <div
-        class="v2-title-3-text text-left v2-canopas-gradient-text reason-text mt-5"
+        v-if="currentRoutePath == '/portfolio/' + id ? 'showCopyrightDiv' : ''"
```
Comment (cp-shruti-s) [inline vue-frontend/src/components/home-new/CTASection.vue:82]: This v-if condition is needed ??

Comment (cp-priyanka-g) [reply vue-frontend/src/components/home-new/CTASection.vue:82]: yes, in order to hide copyright div when we are on home page and show when redirect to portfolio page

Comment (cp-shruti-s) [reply vue-frontend/src/components/home-new/CTASection.vue:82]: we can also manage that condition in below class binding css

Comment (cp-priyanka-g) [reply vue-frontend/src/components/home-new/CTASection.vue:82]: Yesterday i have done that using visibility: hide class in binding but you told to hide that whole div that's why I have used v-if. Should I change it again?

File: vue-frontend/src/components/home-new/CTASection.vue [hunk 5]
```diff
@@ -1,78 +1,107 @@
...
+          </li>
+        </ul>
       </div>
       <div
-        class="v2-title-3-text text-left v2-canopas-gradient-text reason-text mt-5"
+        class="tw-text-center ..."
+        :class="[
+          currentRoutePath == '/portfolio/' + id
+            ? ' tw-pt-16 tw-pb-32 sm:tw-pb-12 xl:tw-pb-1'
+            : 'hide-copyright-text',
```
Comment (cp-shruti-s) [inline vue-frontend/src/components/home-new/CTASection.vue:79]: Can you not used directly css property over here ??

**Review by cp-shruti-s — DISMISSED** [review]

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**